addons/idporten.md

```diff
@@ -0,0 +1,339 @@
+---
+description: >
+  Provisioning and configuration of accompanying ID-porten clients for authentication and authorization in public facing 
+  web applications.
+---
+
+# ID-porten Clients
+
+An accompanying ID-porten Client can be automatically provisioned to your NAIS application. 
+
+The ID-porten Client will be configured with sane defaults to enable usage in both authentication and/or authorization 
+for public facing web applications.
+
+[Digdirator] generates a Kubernetes Secret containing the values needed for your application to integrate ID-porten, 
+e.g. credentials and URLs. The secret will automatically be mounted to the pods of your application during deploy.
+
+Every deploy will trigger rotation of credentials, invalidating any jwk´s that are not in use. 
```

Comment:
Dette betyr at man egentlig har to jwk som man kan bruke samtidig? Er kanskje ikke et stort problem.

Reply:
Ja, de er sant, de er for å ikke få nedetid ved ny deploy. Dette ryddes vekk av digdirator ved neste deploy hvis en pod har kommet i enn annen state, hvis de er de du tenker på?

Reply:
Mhm, men er det noen krise om en app ikke deployes på et halv år? For noe apper blir ikke deploet så ofte. Så da har man potensielt et token som lever veldig lenge, som ikke er i bruk.

Reply:
tror ikke de er en krise nei.